cmd/autoheal/server.go

```diff
@@ -81,25 +81,28 @@ func serverRun(cmd *cobra.Command, args []string) {
 	// Load the Kubernetes configuration:
 	var config *rest.Config
 	_, err = os.Stat(serverKubeConfig)
-	if os.IsNotExist(err) {
-		glog.Infof(
-			"The Kubernetes configuration file '%s' doesn't exist, will try to use the "+
-				"in-cluster configuration",
-			serverKubeConfig,
-		)
-		config, err = rest.InClusterConfig()
+	file, err := os.Open(serverKubeConfig) // For read access.
```

Comment:
I think that using all the built-in functions ( like we used `IsNotExist` before ), and adding a take all `err != nil` at the end is nicer  ... but not a blocker :-)
```
func IsNotExist(err error) bool
func IsPermission(err error) bool
func IsTimeout(err error) bool
```